### PR TITLE
rewrite part that describes ConfigureSEMM script

### PR DESCRIPTION
**Pull request by MaratMussabekov**
issue #4499

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md [hunk 1]
```diff
@@ -103,39 +103,45 @@ The sample scripts include examples of how to set Surface UEFI settings and how
...
+Replace the **FabrikamSEMMSample.pfx** value for the **$certName** variable with the name of your SEMM Certificate file on line 58. The script will create a working directory (named Config) in the folder where your scripts are located, and will then copy the certificate file to this working directory.
 
-Replace the **FabrikamSignerProvisioningPackage.pkg** and **FabrikamUniversalResetPackage.pkg** values on lines 63 and 64 to define the **$ownerPackageName** and **$resetPackageName** variables with your desired names for the SEMM configuration and reset packages. These packages will also be created in the Config directory and hold the configuration for Surface UEFI settings and permissions generated by the script.
+Owner pakage and reset pakage will also be created in the Config directory and hold the configuration for Surface UEFI settings and permissions generated by the script.
```
Comment (nenonix) [inline devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md]:
```suggestion
Owner package and reset package will also be created in the Config directory and hold the configuration for Surface UEFI settings and permissions generated by the script.
```

File: devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md [hunk 2]
```diff
@@ -204,69 +211,169 @@ You can find information about the available settings names and IDs for Surface
...
 
 You can find information about the available settings names and IDs for Surface UEFI in the [Settings Names and IDs](#settings-names-and-ids) section later in this article.
 
 ### Settings registry key
 
-To identify enrolled systems for Configuration Manager, the ConfigureSEMM.ps1 script writes a registry key that can be used to identify enrolled systems as having been installed with the SEMM configuration script. This key can be found at the following location:
+To identify enrolled systems for Configuration Manager, the ConfigureSEMM.ps1 script writes a registry keys that can be used to identify enrolled systems as having been installed with the SEMM configuration script. These keys can be found at the following location:
```
Comment (JohanFreelancer9) [inline devices/surface/use-system-center-configuration-manager-to-manage-devices-with-semm.md]:
```suggestion
To identify enrolled systems for Configuration Manager, the ConfigureSEMM.ps1 script writes registry keys that can be used to identify enrolled systems as having been installed with the SEMM configuration script. These keys can be found at the following location:
```

**Comment by MaratMussabekov** [comment]
Dear @nenonix, @JohanFreelancer9 
your recommendations were applied,
thank you!

**Comment by mypil** [comment]
@Dansimp - PR has been copy edited and is ready for final review. This is based on issue #4499 

Thank you.

**Pull request merged**